controllers/arm.py

```diff
@@ -79,16 +79,18 @@ class ArmController(StateMachine):
     arm_component: Arm
 
     def __init__(self) -> None:
-        self._target_setpoint: Setpoint = Setpoints.STOW
+        self._target_setpoint: Setpoint = Setpoint(0, 0)
         self._about_to_run: bool = False
+        self.initialised = False
 
     def go_to_setpoint(self, setpoint: Setpoint) -> None:
         # If this is a different setpoint, we want to take it
         # Interrupt what we are doing and move to the new setpoint
         if setpoint != self._target_setpoint:
             self._about_to_run = True
+            self.next_state("retracting_arm")
             self._target_setpoint = setpoint
-            self.engage("retracting_arm", force=True)
```

Comment:
If this is the wrong behaviour (I don't think it is) it should be addressed in a separate PR. It's not related to fixing the scoring state machine.

Reply:
Was a mistake made during a rebase, this pr should now not make any changes to the arm